rts/Game/UI/MouseHandler.cpp

```diff
@@ -226,7 +226,8 @@ void CMouseHandler::MouseMove(int x, int y, int dx, int dy)
 {
 	// FIXME: don't update if locked?
 	lastx = std::abs(x);
-	lasty = std::abs(y);
+	// Origin for mousecursor on internal coordinates is top border of view screen
+	lasty = y - globalRendering->viewWindowOffsetY;
```

Comment:
For SDL Mouse coordinates our origin is transposed to the top of main view. This can lead to negative internal mouse coordinate values, which can make it hard to distinguish from off-screen mouse coordinates.

I'm considering an approach to sanitize internal mouse coordinates when off-screen. Honestly `abs(coords)` does not seem suitable and we'd likely want to stay out of the border to not pan the screen indefinitely so vertices and borders should be avoided as well. Probably screenCenter is the best candidate.